tools/memleak.py

```diff
@@ -55,18 +56,25 @@ def run_memleak_test(bench, iterations, report):
         np.sum(rss_peaks[starti+1:] - rss_peaks[starti:-1]) / (endi - starti)))
 
     from matplotlib import pyplot as plt
+    from matplotlib.ticker import EngFormatter
+    formatter0 = EngFormatter(unit='B')
```

Comment:
Why does formatter indexing start at 0, but Axes indexing start at 1?

Reply:
because I copy-pasted the formatter code out of our example :sheep: